src/components/toast/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+type Props = {
+	message?: React.ReactNode
+	dismiss?: () => void
+	className?: string
+}
+
+export function Toast(props: Props) {
+	return (
+		<div 
+			className={`toast show text-white border-0 ${props.className ?? ''}`}
+		>
+			<div className="d-flex">
+				<div className="toast-body text-black">
+					{props.message}
+				</div>
+				<button 
```

Comment:
this should only be visible when dismiss is passed ?? why should u show it when dismiss is not passedd